WebHostLib/models.py

```diff
@@ -56,3 +56,11 @@ class Generation(db.Entity):
     options = Required(buffer, lazy=True)
     meta = Required(LongStr, default=lambda: "{\"race\": false}")
     state = Required(int, default=0, index=True)
+
+
+class GameDataPackage(db.Entity):
+    id = PrimaryKey(int, auto=True)
+    game = Required(str)
+    checksum = Required(str)
```

Comment:
checksum might make a good primary key? Do we even need the game?
This table might be possible as just checksum as primary key and then data in bytes.

Reply:
the use of game is to limit the effects of and have an easier interface to bad seeds being uploaded.

Reply:
oh, do you mean instead of id? could get rid of it, yeah.
not sure how accurate that still is, but there was a time when one wanted to have an integer id to make certain operations easy and fast.